.github/workflows/reusable_on_app_foundation_tag.yaml

```diff
@@ -35,9 +34,6 @@ jobs:
           git tag $tag_name
           git push origin $tag_name
 
-      - name: Make a hotfix branch in this package with an updated version
-        run: mplat admin branch package-hotfix-branch
-
       - name: Commit and push changes
         run: |
           echo "NOTE* If you amend the commit and force push you must have a checkout fetch depth of 0 or at least 2. Otherwise the entire git history will be re-written."
```

Comment:
No need to have a commit here any more right? Looks like all this is now doing is creating and pushing a tag up above.

Reply:
You're absolutely right! Good catch